kolibri/core/discovery/utils/network/connections.py

```diff
@@ -33,7 +33,9 @@ def check_device_info(base_url):
     try:
         info = NetworkClient(base_url=base_url).info
         if info["application"] in ["studio", "kolibri"]:
-            return device_info_defaults.copy().update(info)
+            complete_info = device_info_defaults.copy()
```

Comment:
Seems like it might be simpler just to move the `device_info_defaults` into `kolibri/core/discovery/utils/network/client.py` and apply the change there only. Can then just return `info` here.